### PR TITLE
FIXED opts not being passed into SDK

### LINKED ISSUE #10: Options not being passed into some of the AWS SDK fields
Currently we're passing in "opts", but they aren't being passed into the respective AWS SDK calls.

### PR DESCRIPTION
**Pull request by EwanValentine**
Fixes #10

### CODE REVIEW

File: pkg/pubsub/sns.go [hunk 1]
```diff
@@ -19,17 +19,34 @@ func NewSNSAdapter(client *sns.SNS) *SNSAdapter {
 	return &SNSAdapter{client}
 }
 
+func (sa *SNSAdapter) parseOpts(opts types.Options) map[string]*sns.MessageAttributeValue {
+	atts := make(map[string]*sns.MessageAttributeValue, 0)
+	for key, val := range opts {
+		atts[key] = &sns.MessageAttributeValue{
```
Comment (rickihastings) [inline pkg/pubsub/sns.go]: Could do this here as `types.Options`'s value is an interface, you can just check the type and then assign it directly.

```suggestion
		if attributeValue, ok := val.(*sns.MessageAttributeValue) {
			atts[key] = attributeValue
		}
```

File: pkg/queue/sqs.go [hunk 2]
```diff
@@ -22,12 +22,29 @@ func (qa *SQSAdapter) Queue(service *types.Service, request types.Request) (stri
 	return qa.QueueWithOpts(service, request, types.Options{})
 }
 
+func (qa *SQSAdapter) parseOpts(opts types.Options) map[string]*sqs.MessageAttributeValue {
+	atts := make(map[string]*sqs.MessageAttributeValue, 0)
+	for key, val := range opts {
+		atts[key] = &sqs.MessageAttributeValue{
```
Comment (rickihastings) [inline pkg/queue/sqs.go]: Same as above comment

```suggestion
		if attributeValue, ok := val.(*sqs.MessageAttributeValue) {
			atts[key] = attributeValue
		}
``

**Comment by rickihastings** [comment]
Looks good!

**Comment by EwanValentine** [comment]
Sweet, will merge an bump the version number, probably a patch version, nothing breaking API wise

**Pull request merged**